server/routes/users.js

```diff
@@ -29,7 +29,7 @@ router.get('/profile', isAuthenticated, (req, res) => {
     })
 });
 
-router.put('/users', isAuthenticated, (req, res) => {
+router.put('/profile/edit', isAuthenticated, (req, res) => {
```

Comment:
You changed the wrong one.

Reply:
So this route on the backend should be a PUT to `/users`. You need to update this. 

The route on the frontend should be `/profile/edit` you already have that correct. So don't change anything in your app-routing.module.ts file. I think what you're misunderstanding is that the backend server doesn't care about what frontend page you're showing. So the backend server only cares about data models, in this case, users. So a put to `/users` is how you tell the backend you want to edit a user. 


Now the frontend on the other hand, needs an extra route that basically allows the user to edit their profile. That's why that url path is `/profile/edit` you want to show the user the profile to edit. When the actual edit occurs, you're informing the backend, "Here's the user and what I want to change on that user" by sending a PUT request to `/users`. 

Does that make sense?

Reply:
Ah ok got it